src/js/live-tracker.js

```diff
@@ -43,17 +63,12 @@ class LiveTracker extends Component {
     }
     const liveCurrentTime = this.liveCurrentTime();
     const currentTime = this.player_.currentTime();
-    const seekableIncrement = this.seekableIncrement_;
-
-    // the live edge window is the amount of seconds away from live
-    // that a player can be, but still be considered live.
-    // we add 0.07 because the live tracking happens every 30ms
-    // and we want some wiggle room for short segment live playback
-    const liveEdgeWindow = (seekableIncrement * 2) + 0.07;
```

Comment:
I think removing this could cause problems if there are any live platforms for which the player receives large chunks of content at once, e.g. if it receives 30s of content every 30s.
It would only cause issues if the old `liveEdgeWindow` would be higher then the new `liveTolerance` and I suspect that might never be the case and so this change is fine, but maybe worth checking.

Reply:
I don't think there's any case where the old `liveEdgeWindow` would be higher then the new `liveTolerance` so I think it should be fine? The up-to-date version of the library also removed this calculation and has just replaced it with checking that it's not above the `liveTolerance` value so I don't think there should be any issues.